project.clj

```diff
@@ -3,12 +3,33 @@
   :url "http://funcool.github.io/httpurr"
   :license {:name "Public Domain" :url "http://unlicense.org"}
   :source-paths ["src"]
-  :dependencies [[org.clojure/clojure "1.8.0" :scope "provided"]
-                 [org.clojure/clojurescript "1.9.225" :scope "provided"]
-                 [aleph "0.4.1" :scope "provided"]
-                 [org.clojure/test.check "0.9.0" :scope "test"]
-                 [funcool/promesa "1.5.0"]]
+  :dependencies
+    [[aleph "0.4.1" :scope "provided"]
```

Comment:
I just sorted the dependencies here, nothing special. Do I even need `lein-doo`? I'm not even sure.